README.md

```diff
@@ -1,8 +1,24 @@
+[![Build Status](https://travis-ci.com/NCI-GDC/gdcdatamodel.svg?branch=develop)](https://travis-ci.com/NCI-GDC/gdcdatamodel)
```

Comment:
travis url should be: https://travis-ci.org/NCI-GDC/gdcdatamodel. we might need to do something about the one at travis-ci.com

Reply:
Thanks, and I am not sure should I monitor develop branch or master branch?